### PR TITLE
54l15 partition manager rom start offset

### PR DESCRIPTION
**Pull request by michalek-no**

### CODE REVIEW

**Review by nvlsianpu — APPROVED** [review]

File: boards/arm/nrf54l15pdk_nrf54l15/Kconfig.defconfig [hunk 1]
```diff
@@ -11,6 +11,7 @@ config BT_CTLR
 
 config ROM_START_OFFSET
 	hex
```
Comment (gchwier) [inline boards/arm/nrf54l15pdk_nrf54l15/Kconfig.defconfig:13]: definition type can be removed here

**Review by gchwier — APPROVED** [review]
No rolling reboot, tested with with MCUboot as a ChildImage:
`west build -p -b nrf54l15pdk_nrf54l15_cpuapp ${ZEPHYR_BASE}/samples/hello_world -d build-54l-hello -- -DCONFIG_BOOTLOADER_MCUBOOT=y`
and with Sysbuild:
`west build --sysbuild -p -b nrf54l15pdk_nrf54l15_cpuapp ${ZEPHYR_BASE}/samples/hello_world -d build-54l-hello_sysbuild -- -DSB_CONFIG_BOOTLOADER_MCUBOOT=y`
In both cases works good

File: boards/arm/nrf54l15pdk_nrf54l15/Kconfig.defconfig [hunk 2]
```diff
@@ -10,7 +10,7 @@ config BT_CTLR
 	default BT
 
 config ROM_START_OFFSET
-	hex
```
Comment (sigvartmh) [inline boards/arm/nrf54l15pdk_nrf54l15/Kconfig.defconfig:13]: Why is the type hint removed?

Comment (gchwier) [reply boards/arm/nrf54l15pdk_nrf54l15/Kconfig.defconfig:13]: suggested in the Upstream PR: https://github.com/zephyrproject-rtos/zephyr/pull/68393#discussion_r1520950055

**Pull request merged**